packages/api/src/utility.ts

```diff
@@ -151,5 +152,22 @@ export function capitalizeFirstLetter(str: string): string {
   return str.charAt(0).toUpperCase() + str.slice(1)
 }
 
+function countRichtextChars(blocksCharLength: number, nodes: any) {
+  return nodes.reduce((charLength: number, node: any) => {
+    if (!node.text && !node.children) return charLength
+    if (node.text) {
+      return charLength + (node.text as string).length
+    }
+    return countRichtextChars(charLength, node.children)
+  }, blocksCharLength)
+}
+
+export function validateCommentLength(text: RichTextNode[]) {
```

Comment:
i don't think you need an extra function for this. You can just directly compare the `countRichtextChars` return value to the comment length limit where you call this.